main.go

```diff
@@ -170,7 +173,7 @@ func main() {
 	flag.IntVar(&listenPort, "port", 3000, "Port to listen for webhooks")
 	flag.StringVar(&recipient, "recipient", "", "Keybase user or team#channel to send message to")
 	flag.DurationVar(&interval, "interval", 10*time.Second, "The interval at which to check for watchdog expiry")
-	flag.DurationVar(&expiry, "expiry", 2*time.Minute, "The amount of time after which a non-pinging watchdog check will be considered to have expired")
+	flag.DurationVar(&expiry, "expiry", 10*time.Minute, "The amount of time after which a non-pinging watchdog check will be considered to have expired")
```

Comment:
Reconfigure it in our cluster instead of changing it here, we're just fixing up a public repo here.

Reply:
ye i agree it shouldn't be contained into this PR, but there is an explanation: the prometheus' watchdog is kinda strange behaving, since it sends watchdog notifications completely irregularly. even though i've decreased the delay between sending alerts to 1 second, it happens that following notification are sent within seconds, but after a moment within minutes. unfortunately ive failed in finding the result of that behavior.